### ISSUE TITLE
Job templates

### ISSUE THREAD

**Issue by kaitj** [pvandyken/kslurm#20]
Not necessarily `kslurm` specific and all minor:
_First two points was just looking through gh_  

1. I noticed this when I was working `neuroglia-helpers` as well, but there may be redundant job templates, such as `32core128gb3h` vs `ShortFat` (see slurm_job_templates.json). Wonder if it might be better to move away from the original implementation of naming them (e.g. ShortFat) to just naming by resource (e.g. 32core128gb24h).

2. `32core128gb12h` is given 2 cpus rather than 32 in the current `slurm_job_templates.json`  

3. This one might be on my end, but the `data` folder is missing on installation. When running `krun --list`, a `FileNotFoundError` is returned with it being unable to local the `slurm_job_templates.json`

**Comment by pvandyken** [comment]
I agree with the first point, the `32core...` type syntax is redundant anyway, far easier to just explicitly call e.g. `krun 32 128G 12:00`

Do you have the full trace for point 3? Can't reproduce on my machine

**Comment by pvandyken** [comment]
Just realize our conclusions on which duplicates to remove were opposite each other. @akhanf, did you originally create the templates? Do you have an opinion on this?

**Comment by kaitj** [comment]
Re: templates - Either method of naming would work (if `--list` also lists the resources). My preference was towards the `128gb32core...` type of naming since it is immediately clear what resources are requested, but it is a bit wordier to type out.

**Comment by pvandyken** [comment]
Just fixed point 3 on `v0.3.5`
